org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/XMLCompletions.java

```diff
@@ -498,7 +498,7 @@ public AutoCloseTagResponse doTagComplete(DOMDocument xmlDocument, Position posi
 			}
 		} else if (cBefore == '<' && c == '/') { // Case: <a> </|
 			DOMNode node = xmlDocument.findNodeBefore(offset);
-			while (node != null && node.isClosed()) {
+			while ((node != null && node.isClosed()) || (node.isElement() && ((DOMElement) node).isOrphanEndTag())) {

```

Comment:
I think you are planning on addressing this in the other PR, but:

```xml
<a></</a>
```
The code action puts:
```xml
<a></a</a>
```

Reply:
Please create an issue about this usecase and please suggest what you would to generate.

Reply:
Okay. It make sense to me to move this to a separate issue. I opened https://github.com/eclipse/lemminx/issues/889